modules/stanford_profile_styles/lib/scss/component/node/stanford_opportunity.scss

```diff
@@ -321,42 +349,44 @@
       p {
         margin-bottom: 50px;
       }
-    }
 
-    h3 {
-      @include type-f;
+      h3 {
+        @include type-f;
 
-      margin-bottom: 0;
-      word-break: break-word;
-      color: $su-color-black;
-
-      &.su-link--action {
+        margin-bottom: 0;
+        word-break: break-word;
         color: $su-color-black;
 
-        &:hover,
-        &:focus {
-          color: $su-color-digital-red;
-          text-decoration: underline;
-        }
+        &.su-link--action {
```

Comment:
is this a possibility? `<h3>` with `su-link--action` class?

Reply:
On a teaser for an Opportunity ct, I couldn't change the heading. I got an `h2` for the teaser headline when rendered. I got an `h3` inside the opportunity card. 

On a list of Opportunities, I also couldn't change the heading type. I got `h2` for the list headline and an `h3` inside the opportunity card.

On a card paragraph, I could select different heading types.